modules/installation-azure-user-defined-routing.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-private.adoc
+
+[id="installation-azure-user-defined-routing_{context}"]
+= User-defined outbound routing
+
+In {product-title}, you can choose your own outbound routing for a cluster to
+connect to the internet. This allows you to skip the creation of public IP
+addresses and public load balancers. This is required if you want to prevent
+exposing external endpoints for a private cluster.
```

Comment:
> This is required if you want to prevent exposing external endpoints for a private cluster.

Even in case of private clusters with loadbalancer outbound type (the default), the endpoints are not exposed publicly. public IPs exists but there is no ingress allowed or configured on those. UDR allows the users to not the public IPs etc. in additions to no external endpoints.. 

hopefully that is making it clearer. So I think we can maybe drop this line here.

Reply:
Thanks for explaining that. I didn't fully understand that connection. I'll remove that line and adjust the rest of the text as necessary.